files/.babelrc

```diff
@@ -0,0 +1,18 @@
+{
+  "presets": [
+    [
+      "env",
```

Comment:
babel-preset-env is more flexible than babel-preset-es2015 and I'm thinking probably a good fit as the default for @glimmer/application-pipeline.